training/src/anemoi/training/train/train.py

```diff
@@ -278,6 +278,8 @@ def tensorboard_logger(self) -> pl.loggers.TensorBoardLogger:
     def _get_warm_start_checkpoint(self) -> Path | None:
         """Returns the warm start checkpoint path if specified."""
         warm_start_path = self.config.system.input.warm_start
+        if isinstance(warm_start_path, str):
```

Comment:
@timothyas thanks for catching this.  Would we need the if statement? you could just simply do:
```
warm_start_path = Path(self.config.system.input.warm_start)
```

Reply:
Yeah that makes sense to me, I'll get rid of the if statement. I guess I was being overly cautious